src/modules/homepage.js

```diff
@@ -0,0 +1,54 @@
+const getData = async () => {
+  const request = await fetch('https://api.tvmaze.com/shows');
+  const response = await request.json();
+  return response;
+};
+
+const render = async () => {
+  const container = document.querySelector('.whole-container');
+  const jsonData = await getData();
+  let item = '';
+  for (let i = 0; i < 6; i++) {
+    item += `
+      <div class="item">
+          <img src="${jsonData[i].image.medium}" alt="" />
+          <div class="details">
+            <p>${jsonData[i].name}</p>
+            <div>
+                <i id="${jsonData[i].id}" class="fa-regular fa-heart"></i>
+                <div class="likesContainer"></div>
+            </div>
+          </div>
+          <button class="comment-btn" id="btn-${jsonData[i].id}">Comments</button>
+          <button class="${i}">Reservations</button>
+        </div>`;
+  }
+  container.innerHTML = item;
+};
```

Comment:
[OPTIONAL] In order to improve performance, when rendering a large number of items 📦 , it's better to use `document.createDocumentFragment()` instead of appending elements to the `DOM` one at a time. 🤓  This can help improve performance. ⚡ It could go like this:
```
const renderShows = (shows) => {
  const fragment = document.createDocumentFragment();
  shows.forEach((show) => {
    const item = document.createElement('div');
    item.classList.add('item');
    item.innerHTML = `
      <img src="${show.image.medium}" alt="" />
      <div class="details">
        <p>${show.name}</p>
        <div>
          <i id="${show.id}" class="fa-regular fa-heart"></i>
          <div class="likesContainer"></div>
        </div>
      </div>
      <button class="comment-btn" id="btn-${show.id}">Comments</button>
      <button class="${show.id}">Reservations</button>
    `;
    fragment.appendChild(item);
  });
  const container = document.querySelector('.whole-container');
  container.appendChild(fragment);
};
```
Keep in mind that this is just an example, and it might need some adaptation 😉